### ISSUE TITLE
about tfrecords

### ISSUE THREAD

**Issue by luoruisichuan** [ycszen/tf_lab#2]
受益匪浅！有几点问题，再请教下：
1、“关于Example Feature的相关定义和详细内容，我推荐去官网查看相关API”，请问有相关的API文档资料或网页吗？我一直没找到；
2、如果我们以TFRecords文件的形式读入图片数据进行模型训练获得模型后，新的图片数据，要对其进行分类，这时候这1张新的图片数据，需要以什么方式读入模型呢？有没有相应的脚本学习下呢？
3、“
之后我们可以在训练的时候这样使用
img, label = read_and_decode("train.tfrecords")
# 使用shuffle_batch可以随机打乱输入

img_batch, label_batch = tf.train.shuffle_batch([img, label],
                                                batch_size=30, capacity=2000,
                                                min_after_dequeue=1000)
”
我猜测  img, label = read_and_decode("train.tfrecords")  让我们获得了图片及其类别标签（因为是多张图片，股都是链表的形式)
"
img_batch, label_batch = tf.train.shuffle_batch([img, label],
                                                batch_size=30, capacity=2000,
                                                min_after_dequeue=1000)
"
这句我猜测是将链表形式的图片数据及其标签，分成多个batch用于后续输入训练模型的语句，那么有个问题，batch_size=30意思是一个batch有多少个样本，capacity=2000和                                            min_after_dequeue=1000，分别是什么意思呢？
多谢指教哈！

**Comment by yu-changqian** [comment]
1.https://www.tensorflow.org/versions/r0.9/api_docs/python/io_ops.html#example-protocol-buffer
2.测试的时候输入图片可以是原生图片，只要保证shape一致就行，但是还是推荐使用tfrecord，因为便于使用线程和队列来加载图片，更为高效。之后我可能会写一个测试的demo
3.batch_size是一个batch有多少张图片，capacity控制队列里元素的最大数量; min_after_dequeue控制队列里元素在dequeue后的最小数量
